### PR TITLE
Order of activity

### PR DESCRIPTION
**Pull request by Keshav7104**
## Summary
Solved the issue of unordered activity option in submit page sorted the categories in helpers simply will solve the problem
Screenshot (76)

…
----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included

### CODE REVIEW

**Comment by Keshav7104** [comment]
due to some internet issue, 2 commits were not reflected on the GitHub so kindly consider these commits too

File: helpers/formSchema.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const  schema = {
     '0': [],
     // Memorandum of Understanding (MoU)
-    '1': [
+    '10': [
```
Comment (tripathics) [inline helpers/formSchema.js:4]: **Bug**
Don't change keys. Also since you've changed the keys for form schema only and left the previewTemplate unchanged, they don't sync up now

File: components/form/form-fc.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ const FormFC = ({
 
   const categoriesSelect = Categories.map((category, i) => {
     if (i === 0) {
-      return <option key={i} value={i}>Select an activity category</option>
+      return <option key={i} value={i} hidden>Select an activity category</option>
```
Comment (tripathics) [inline components/form/form-fc.js:28]: Unrelated change

File: components/form/Form.module.scss [hunk 3]
```diff
@@ -50,6 +50,23 @@ $form-muted: #ced4da;
...
+        width: 1.8rem;
+        height: 2rem;
+        background-image: url("data:image/svg+xml,%3Csvg fill='%23000000' version='1.1' id='Capa_1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' viewBox='0 0 94.926 94.926' xml:space='preserve'%3E%3Cg id='SVGRepo_bgCarrier' stroke-width='0'%3E%3C/g%3E%3Cg id='SVGRepo_tracerCarrier' stroke-linecap='round' stroke-linejoin='round'%3E%3C/g%3E%3Cg id='SVGRepo_iconCarrier'%3E%3Cg%3E%3Cpath d='M55.931,47.463L94.306,9.09c0.826-0.827,0.826-2.167,0-2.994L88.833,0.62C88.436,0.224,87.896,0,87.335,0 c-0.562,0-1.101,0.224-1.498,0.62L47.463,38.994L9.089,0.62c-0.795-0.795-2.202-0.794-2.995,0L0.622,6.096 c-0.827,0.827-0.827,2.167,0,2.994l38.374,38.373L0.622,85.836c-0.827,0.827-0.827,2.167,0,2.994l5.473,5.476 c0.397,0.396,0.936,0.62,1.498,0.62s1.1-0.224,1.497-0.62l38.374-38.374l38.374,38.374c0.397,0.396,0.937,0.62,1.498,0.62 s1.101-0.224,1.498-0.62l5.473-5.476c0.826-0.827,0.826-2.167,0-2.994L55.931,47.463z'%3E%3C/path%3E%3C/g%3E%3C/g%3E%3C/svg%3E");
+        background-position: center;
+        background-size: 2em;
+        background-repeat: no-repeat;
+        content: "";
+        background-color: #f1f1ea;
+    }
```
Comment (tripathics) [inline components/form/Form.module.scss:68]: Unrelated change to the issue. But still appreciate the good practice of following the codebase while writing

File: helpers/helpers.js [hunk 4]
```diff
@@ -21,9 +21,9 @@ const Categories = [
   'Outreach Activity',
   'Announcement',
   'Awards',
-  // 'Other'
+  //others
 ]
-
+Categories.sort();
```
Comment (tripathics) [inline helpers/helpers.js:26]: Even though the approach is introducing bugs, I want to bring to your attention that you sorted Categories but left CategoryTitles.
